src/travis2docker/templates/build.sh

```diff
@@ -79,6 +79,19 @@ install_dev_tools(){
         # byobu \
         # multitail  # Set the terminal with red letters build the docker
 
+    # Node is grossly outdated (v10) in repositories. Manually install it to get current version and make eslint work.
+    sudo apt remove --purge nodejs
+    sudo apt autoremove --purge
+    NODE_DOWNLOAD="/tmp/node-v16.16.0-linux-x64.tar.xz"
```

Comment:
I got it

I understand the point

But it could get big changes for the odoo-bin results

Question, do you know if it is possible to use an virtualenv for vscode

I mean, we could use something similar to pylint-odoo:
 - https://github.com/OCA/pylint-odoo/blob/4960c52cf8cffd52010f65b1caa374c65ad1cd0a/tox.ini#L20

Notice it is using `nodeenv` where you can install the required version but only for this virtualenv without affecting the global one (used by odoo-bin to run unittest)

Reply:
Could you separate the eslint part in another PR, please?

I think the progress as far is good but I can't merge it bc this

Reply:
Yes, done